code/game/objects/items/storage/pouch.dm

```diff
@@ -514,17 +514,23 @@
 	max_storage_space = 14
 	can_hold = list(/obj/item/reagent_containers/hypospray/autoinjector)
 
-/obj/item/storage/pouch/medical_injectors/firstaid
-	desc = "Standard marine first-aid injector pouch. Specialized to store only auto-injectors. Contains basic injectors, a stabilizing injector, stimulant injector, and an emergency injector."
+/obj/item/storage/pouch/medical_injectors/basic/firstaid
```

Comment:
Why do you define the /basic/firstaid then define /basic then initialize /basic then initialize /basic/firstaid? The organization looks and feels weird to. Would it not be better to define /basic then initialize it then define /basic/firstaid and then initialize that too?